### PR TITLE
zh-translation: news/releases/1.9.x/announcing-1.9.6/index.md

### PR DESCRIPTION
**Pull request by orangegzx**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/news/releases/1.9.x/announcing-1.9.6/index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+{{< relnote >}}
+
+##  安全更新{#security-update}
+
+- __[CVE-2021-34824](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-34824)__：
+Istio 包含一个可远程利用的漏洞，在 `Gateway` 和 `DestinationRule` `credentialName` 字段中指定的凭证可以从不同的命名空间访问。有关更多详细信息，请参阅 [ISTIO-SECURITY-2021-007 bulletin](/zh/news/security/istio-security-2021-007)。
+    - __CVSS Score__： 9.1 [CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:C/C:H/I:L/A:L](https://www.first.org/cvss/calculator/3.1#CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:C/C:H/I:L/A:L)
+
+## 变化
```
Comment (kebe7jun) [inline content/zh/news/releases/1.9.x/announcing-1.9.6/index.md:22]: missing ID.

File: content/zh/news/releases/1.9.x/announcing-1.9.6/index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+release: 1.9.6
+aliases:
+    - /zh/news/announcing-1.9.6
+---
+
+此版本修复了在 6 月 24 日发布的帖子 [ISTIO-SECURITY-2021-007](/zh/news/security/istio-security-2021-007) 中描述的安全问题以及一些小漏洞，提高了稳健性。本次发布说明主要描述 Istio 1.9.5 和 1.9.6 版本之间的区别。
+
+{{< relnote >}}
+
+##  安全更新{#security-update}
```
Comment (kebe7jun) [inline content/zh/news/releases/1.9.x/announcing-1.9.6/index.md:16]:
```suggestion
## 安全更新{#security-update}
```

File: content/zh/news/releases/1.9.x/announcing-1.9.6/index.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+- __[CVE-2021-34824](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-34824)__：
+Istio 包含一个可远程利用的漏洞，在 `Gateway` 和 `DestinationRule` `credentialName` 字段中指定的凭证可以从不同的命名空间访问。有关更多详细信息，请参阅 [ISTIO-SECURITY-2021-007 bulletin](/zh/news/security/istio-security-2021-007)。
+    - __CVSS Score__： 9.1 [CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:C/C:H/I:L/A:L](https://www.first.org/cvss/calculator/3.1#CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:C/C:H/I:L/A:L)
+
+## 变化
+
+- **修复** 当使用 `traffic.sidecar.istio.io/includeOutboundPorts` 注释时，IPv6 的 Iptable 规则不正确的问题。
```
Comment (kebe7jun) [inline content/zh/news/releases/1.9.x/announcing-1.9.6/index.md:24]: iptables 是专有名词，不需要去掉s。

**Review by kebe7jun — APPROVED** [review]

**Comment by hzxuzhonghu** [comment]
1.9 is not maintained ， why not translate docs for master

**Pull request merged**